### PR TITLE
Simplified user_activity loading into vertica

### PR DESCRIPTION
**Pull request by HassanJaveed84**
@mulby @brianhw 
test_internal_reporting_user_activity acceptance test breaks because of the introduction of ExternalURL as the dependency. How can I set that up in the test to make it work ?

### CODE REVIEW

File: edx/analytics/tasks/load_internal_reporting_user_activity.py [hunk 1]
```diff
@@ -224,3 +220,24 @@ def requires(self):
                 history_schema=self.history_schema
             )
         ]
+
+
+class UserActivityWorkflow(luigi.WrapperTask):
```
Comment (HassanJaveed84) [inline edx/analytics/tasks/load_internal_reporting_user_activity.py:225]: Should this be moved to user_activity.py ?

**Comment by HassanJaveed84** [comment]
@mulby updated acceptance test.

File: edx/analytics/tasks/load_internal_reporting_user_activity.py (hunk 1 above)
Comment (brianhw) [inline edx/analytics/tasks/load_internal_reporting_user_activity.py:225]: Good question.  I don't think it hugely matters, but I'm inclined to say 'no'.  This is the work that needs to be run every week, so putting it with the other weekly code might make sense.  But this file is dependent on the other file anyway, and having the other file be dependent on this one as well seems unnecessarily circular.  I would also put the workflow that does the daily writing to Vertica in this file, not in user_activity.

**Comment by brianhw** [comment]
LGTM. :+1:

File: edx/analytics/tasks/load_internal_reporting_user_activity.py (hunk 1 above)
Comment (mulby) [inline edx/analytics/tasks/load_internal_reporting_user_activity.py:225]: I'm a little confused, when will these be called? Is it the new entry point?

If so, what is happening with InternalReportingUserActivityWorkflow?

Comment (brianhw) [inline edx/analytics/tasks/load_internal_reporting_user_activity.py:225]: I understand this to be the new entry point for a weekly job.  Then there is a separate daily job to write this output of the weekly job to Vertica.

Comment (mulby) [inline edx/analytics/tasks/load_internal_reporting_user_activity.py:225]: Ah, that makes sense.

**Comment by mulby** [comment]
:+1:

**Pull request merged**